Dockerfile

```diff
@@ -16,6 +16,6 @@ COPY --from=builder /go/src/github.com/stolostron/cluster-proxy-addon/cluster-pr
 COPY --from=builder /go/src/github.com/stolostron/cluster-proxy-addon/proxy-agent /
 COPY --from=builder /go/src/github.com/stolostron/cluster-proxy-addon/proxy-server /
 
-RUN microdnf update && microdnf clean all
+RUN microdnf update -y && microdnf clean all
```

Comment:
That's a very good point! I changed this because the build process stuck and wait for input `y/N`. But it only occured once, and I can't reproduce it.

Reply:
The change is reverted, please take another look! Thanks